uvm/servlets/admin/config/network/view/Interfaces.js

```diff
@@ -12,6 +12,15 @@ Ext.define('Ung.config.network.view.Interfaces', {
         padding: '8 5',
         style: { fontSize: '12px' },
         html: '<strong>' + 'Interface configuration'.t() + '</strong> <br/>' +  'Use this page to configure each interface\'s configuration and its mapping to a physical network card.'.t()
+    }, {
+        xtype: 'tbtext',
+        padding: '0 5',
+        style: { fontSize: '12px' },
+        hidden: true,
+        bind: {
+            hidden: '{allowAddInterfaces}'
+        }, 
+        html: '<br/><strong style="color:#FF0000">' + 'Maximum VLAN Interfaces reached.'.t() + '</strong>'
```

Comment:
While technically true that it applies to VLANs, I think it should just be "Maximum interfaces reached" to drill home that "no more interfaces of any kind".

Reply:
This is outdated, resolving